tests/test_dir.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
+import unittest
+from jnius.reflect import autoclass
+
+class DirTest(unittest.TestCase):
+
+    def test_varargs_dir(self):
+        # >>> from jnius import autoclass
+        # >>> cls = autoclass('java.lang.System')
+        # >>> dir(cls.out.printf)
+        # [(['java/lang/String', 'java/lang/Object...'], 'java/io/PrintStream'),
+        # (['java/util/Locale', 'java/lang/String', 'java/lang/Object...'], 'java/io/PrintStream')]
```

Comment:
why not test that this is true, in the test? maybe
```
assert (['java/lang/String', 'java/lang/Object...'], 'java/io/PrintStream') in dir(cls.out.printf)
```

etc?

Reply:
I thought matching exact strings are miscellaneous but it seems more appropriate than just plain comments.